contracts/eosio.system/src/producer_pay.cpp

```diff
@@ -72,10 +72,12 @@ namespace eosiosystem {
 
       if( usecs_since_last_fill > 0 && _gstate.last_pervote_bucket_fill > time_point() ) {
          auto new_tokens = static_cast<int64_t>( (continuous_rate * double(token_supply.amount) * double(usecs_since_last_fill)) / double(useconds_per_year) );
+         const uint64_t days_in_gbm_period = useconds_in_gbm_period / useconds_per_day;
+         const uint64_t days_remaining_in_gbm_period = (gbm_final_time - current_time_point()).count();
          // needs to be 1/2 Savings, 1/6 Producers, 1/6 Voters, 1/6 Genesis Block Member
```

Comment:
Ahh - something was left out in the descrition. We would also want to change the continuous rate to 5%: static constexpr double continuous_rate = 0.04879; // 5% annual rate here: https://github.com/worldwide-asset-exchange/wax-system-contracts/blob/develop/contracts/eosio.system/include/eosio.system/eosio.system.hpp#L62

and correspondingly, divide by 5 for the to_voters variable in the line 78 below. We should adjust this comment accordingly and maybe say that "gbm receives a linearly deflating share over three years"